libsolidity/interface/StandardCompiler.cpp

```diff
@@ -337,16 +339,30 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 			return formatFatalError("JSONError", "library entry is not a JSON object.");
 		for (auto const& library: jsonSourceName.getMemberNames())
 		{
+			string const& address = jsonSourceName[library].asString();
```

Comment:
Need to double check `asString`.

Reply:
does `asString()` really return a reference?

Reply:
No, it actually creates a new instance in some cases.